### ISSUE TITLE
Datapackage validator uses old schema for specification

### ISSUE THREAD

**Issue by Mikanebu** [frictionlessdata/datapackage-php#36]
I tried to use this data package `https://datahub.io/core/country-list/datapackage.json`, but it throws error on `sources` property 
```
[sources[0].uri] The property uri is required, [sources[0].name] Does not match the regex pattern ^([-a-z0-9._/])+$'
```
Please, could you update with new spec https://specs.frictionlessdata.io/?

**Comment by OriHoch** [comment]
thanks, I plan to update to v1 specs by Nov. 30

I'm very interested in use cases of this library, if you could elaborate on your use case it will be very helpful

**Comment by OriHoch** [comment]
@Mikanebu - fixed (merged to master)
please pay attention that url you provided is not a tabular data package (profile is `data-resource` and not `tabluar-data-resource` - so it will yield the rows as strings, not as arrays, this is according to the spec

**Comment by OriHoch** [comment]
published `v0.1.7`
